src/engine/renderer/glsl_source/generic2D_fp.glsl

```diff
@@ -0,0 +1,26 @@
+/*
+===========================================================================
+Copyright (C) 2006-2011 Robert Beckebans <trebor_7@users.sourceforge.net>
```

Comment:
I forgot to remove this when I stripped it down to a define, but a define can't have a license, the whole header can be deleted. 🤭️

Reply:
Done.

Reply:
I still see the jumbo header, maybe the change was not added when committing?

Reply:
oops